apis/control/v1alpha1/k8ssandratask_types.go

```diff
@@ -199,3 +175,47 @@ func getConditionStatus(s cassapi.CassandraTaskStatus, conditionType cassapi.Job
 func init() {
 	SchemeBuilder.Register(&K8ssandraTask{}, &K8ssandraTaskList{})
 }
+
+// getCondition converts a primitive boolean into a k8s ConditionStatus
+func getCondition(condition bool) corev1.ConditionStatus {
+	if condition {
+		return corev1.ConditionTrue
+	}
+	return corev1.ConditionFalse
+}
+
+// getGlobalStatus returns an aggregate status of the Datacenters task statuses
+func getGlobalStatus(t K8ssandraTaskStatus) globalStatus {
+	status := globalStatus{
```

Comment:
Really nitpicking here but can we name this `globalStatus`?
We are dealing with two statuses in this method, "DC-level status" vs global one, the naming should make it clear which is which.